src/Controller/GmoLinkTypePlusController.php

```diff
@@ -109,7 +149,6 @@ public function responseProcessor(Request $request) {
     }
     catch (\Exception $e) {
       $this->loggerFactory->error($e->getMessage());
-      return new JsonResponse(1);
     }
```

Comment:
We need this to send the response for webhook / completeUrl. Have to verify this